management/server/grpcserver.go

```diff
@@ -109,17 +112,52 @@ func (s *GRPCServer) GetServerKey(ctx context.Context, req *proto.Empty) (*proto
 	}, nil
 }
 
+// extractIps returns real peer ip and grpc peer address (ip:port)
+func extractIps(ctx context.Context) (string, string) {
```

Comment:
Do we need 2 addresses here? Maybe just getting from the headers and using gRPCPeer.FromContext as a fallback in case headers are not available?

Reply:
To keep backward compatibility as previously we were using the `ip:port` of GRPC peer. If the port is not needed I'm happy to keep only one.

Reply:
Got it, we can remove, it was used only in debug logs.